### PR TITLE
feat(appoverview): add change documents sub menu

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Added sub-menu for change documents for active apps

## Why

To enhance active apps

## Issue

Ref: CPLP-3084

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: src/components/pages/AppOverview/ChangeDocuments.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+export default function ChangeDocuments() {
+  return <>In Progress</>
```
Comment (manojava-gk) [inline src/components/pages/AppOverview/ChangeDocuments.tsx:21]: Can you move this to shared folder and make it more reusable? better name for the file

**Review by manojava-gk — COMMENTED** [review]
small input

File: src/components/pages/AppOverview/ChangeDocuments.tsx (hunk 1 above)
Comment (lavanya-bmw) [reply src/components/pages/AppOverview/ChangeDocuments.tsx:21]: This feature is new and I don't think this needs to be in shared components

Comment (manojava-gk) [reply src/components/pages/AppOverview/ChangeDocuments.tsx:21]: concern is not to hard code any text and give a common name to the file. We might see another text in place of `IN PROGRESS` in future. Current file name and the content is totally different

**Review by oyo — APPROVED** [review]

**Pull request merged**